### PR TITLE
Supersedes #216

### PR DESCRIPTION
**Pull request by CyberFlameGO**
Adds the config option, and since I don't know how to make suggestions within GitHub to a file that isn't changed in a PR, I made this PR.

### CODE REVIEW

**Comment by CyberFlameGO** [comment]
@Elierrr I'm pretty sure I did the configuration correctly, I just went off of how it was done on the NachoConfig file (and one of the changed files from your PR) and Copilot's suggestions, let me know if I made any mistakes.

@SaithTime Thanks for your contributions, I cloned your repo, checked out your branch, and pushed them to my fork's remote so your contributions + signatures (if applicable, which, in this case will've been your merge commit from GitHub) have been retained (in retrospect, I could've just messed around with making a patch branch by making a useless README edit and resetting the commit + force-pushing it, because I already tried PR'ing your branch to a branch on my fork even without the latest base commit, but this worked anyways).

@Lucaskyy let me know if this is alright, and if so, this is ready to be merged :+1:

**Comment by Lucaskyy** [comment]
LGTM! :)

File: NachoSpigot-Server/src/main/java/me/elier/nachospigot/config/NachoConfig.java [hunk 1]
```diff
@@ -309,4 +309,12 @@ public static void hideProjectilesFromHiddenPlayers() {
         hideProjectilesFromHiddenPlayers = getBoolean("settings.hide-projectiles-from-hidden-players", false);
         c.addComment("settings.hide-projectiles-from-hidden-players", "Hides projectiles from hidden players");
     }
+
+    public static boolean antiEnderPearlGlitch;
+    
+    public static void antiEnderPearlGlitch() {
```
Comment (Elierrr) [inline NachoSpigot-Server/src/main/java/me/elier/nachospigot/config/NachoConfig.java:315]: The method always has to be private static or else it won't be detected. Also make sure to update the configuration version when you make a new config.

**Review by Elierrr — COMMENTED** [review]
@CyberFlameGO Here's some info whenever you make a new config.

**Pull request merged**